src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -0,0 +1,186 @@
+    sql = (
+        "SELECT bus_id, point, voltage, osm_id, subst_name, osm_www FROM "
+        + cfg["sources"]["hvmv_subs"]
+    )
+    substations_mvhv = gpd.GeoDataFrame.from_postgis(
+        sql, con, crs="EPSG:4326", geom_col="point"
+    )
+
+    # Create columns for bus_id, geometry and osm_web in the offshore df
+    offshore["bus_id"] = np.nan
+    offshore["geom"] = Point(0, 0)
+    offshore["osm_web"] = ""  # Just for testing purposes
+
+    # Match bus_id, geometry and osm_web
+    for index, wind_park in offshore.iterrows():
+        if (
+            len(
+                substations_ehv[
+                    substations_ehv["osm_id"] == wind_park["osm_id"]
+                ].index
+            )
+            > 0
+        ):
+            a = substations_ehv[
+                substations_ehv["osm_id"] == wind_park["osm_id"]
+            ].index[0]
+            offshore.at[index, "bus_id"] = substations_ehv.at[a, "bus_id"]
+            offshore.at[index, "geom"] = substations_ehv.at[a, "point"]
+            offshore.at[index, "osm_web"] = substations_ehv.at[a, "osm_www"]
+        elif (
+            len(
+                substations_mvhv[
+                    substations_mvhv["osm_id"] == wind_park["osm_id"]
+                ].index
+            )
+            > 0
+        ):
+            a = substations_mvhv[
+                substations_mvhv["osm_id"] == wind_park["osm_id"]
+            ].index[0]
+            offshore.at[index, "bus_id"] = substations_mvhv.at[a, "bus_id"]
+            offshore.at[index, "geom"] = substations_mvhv.at[a, "point"]
+            offshore.at[index, "osm_web"] = substations_mvhv.at[a, "osm_www"]
+        else:
+            print(f'Wind offshore farm not found: {wind_park["osm_id"]}')
+
+    # Drop offshore wind farms without found connexion point
+    offshore.dropna(subset=["bus_id"], inplace=True)
+
+    # Assign voltage levels to wind offshore parks
+    offshore["voltage_level"] = np.nan
+    offshore.loc[
+        offshore[offshore["Spannungsebene in kV"] == 110].index,
+        "voltage_level",] = 3
+    offshore.loc[
+        offshore[offshore["Spannungsebene in kV"] > 110].index,
+        "voltage_level"] = 1
+
+    # Assign static values
+    offshore["carrier"] = "wind_offshore"
+    offshore["chp"] = False
+    offshore["el_capacity"] = offshore["C 2035"]
+    offshore["scenario"] = "eGon2035"
+    offshore["th_capacity"] = 0
+
+    # Delete unnecessary columns
+    offshore.drop(
+        [
+            "Netzverknuepfungspunkt",
+            "Spannungsebene in kV",
+            "C 2035",
+            "osm_id",
+            "osm_web",
+        ],
+        axis=1,
+        inplace=True,
+    )
+
+    # convert column "bus_id" to integer
+    offshore["bus_id"] = offshore["bus_id"].apply(int)
+    offshore["voltage_level"] = offshore["voltage_level"].apply(int)
+
+    # Look for the maximum id in the table egon_power_plants
+    sql = "SELECT MAX(id) FROM supply.egon_power_plants"
```

Comment:
You can also use the target table name from the configuration file here.

Reply:
Done.